.github/tests/federation-bundle-endpoint/post-install.sh

```diff
@@ -0,0 +1,17 @@
+#!/usr/bin/env bash
+
+set -x
+
+SCRIPT=$(readlink -f "$0")
+SCRIPTPATH=$(dirname "$SCRIPT")
+NS="${SCRIPTPATH##*/}"
```

Comment:
NS is the same thing as $scenario. Should probably just use that, or add NS up at the workflow level.